x-pack/plugins/siem/public/management/components/management_page_view.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useParams } from 'react-router-dom';
+import { PageView, PageViewProps } from '../../common/components/endpoint/page_view';
+import { ManagementSubTab } from '../types';
+import { getManagementUrl } from '../common/routing';
+
+export const ManagementPageView = memo<Omit<PageViewProps, 'tabs'>>(options => {
+  const { tabName } = useParams<{ tabName: ManagementSubTab }>();
+  const tabs = useMemo((): PageViewProps['tabs'] => {
+    return [
+      {
+        name: i18n.translate('xpack.siem.managementTabs.endpoints', {
+          defaultMessage: 'Endpoints',
+        }),
+        id: ManagementSubTab.endpoints,
+        isSelected: tabName === ManagementSubTab.endpoints,
+        href: `#${getManagementUrl('endpointList')}`,
+      },
+      {
+        name: i18n.translate('xpack.siem.managementTabs.policies', { defaultMessage: 'Policies' }),
+        id: ManagementSubTab.policies,
+        isSelected: tabName === ManagementSubTab.policies,
+        href: `#${getManagementUrl('policyList')}`,
+      },
+    ];
+  }, [tabName]);
```

Comment:
I noticed that running ESLint with `--fix` no longer auto-updates the dependencies on these types of utilities (like useMemo(), useCallback, etc). Maybe the rule was adjusted in SIEM?

Reply:
it is turned off. https://github.com/elastic/kibana/blob/master/.eslintrc.js#L759

Reply:
@oatkiller that's unfortunate. I feel that causes more issues than it prevents. I will make our team aware of it to ensure they pay more attention to code changes we do in these types of memoizing functions.

@XavierM you have any background on why this is the case? 😬